src/includes/blocks/technology.pug

```diff
@@ -3,7 +3,7 @@
     .container
         h1 {{ page.title }}  (Technology)
         p {{ page.description }}
-        p {{ content }}
+        {{ content }}
```

Comment:
here is a better way to do it
![image](https://cloud.githubusercontent.com/assets/656739/25725787/5e58c0aa-3119-11e7-87ea-9c76cb2e7e89.png)
which creates this html

![image](https://cloud.githubusercontent.com/assets/656739/25725798/6a744a6c-3119-11e7-9db0-1e4f9251c8dc.png)

which looks like created the correct html

![image](https://cloud.githubusercontent.com/assets/656739/25725821/80ccfd40-3119-11e7-9918-68df5d1a7404.png)